code/game/objects/structures/crates_lockers/crates/critter.dm

```diff
@@ -12,9 +12,12 @@
 	close_sound = 'sound/machines/wooden_closet_close.ogg'
 	open_sound_volume = 25
 	close_sound_volume = 50
-	contents_pressure_protection = 0.8
-	var/obj/item/tank/internals/emergency_oxygen/tank
+	contents_pressure_protection = 0.
```

Comment:
wha

Reply:
copy/paste whoopsie, thanks for catching it.

if you're wondering why that compiles, it's because `0.` == `0`
![image](https://github.com/Monkestation/Monkestation2.0/assets/65794972/fb9bfa4e-fb1f-4ad2-abc8-cf668479c138)